app/main.py

```diff
@@ -210,5 +212,12 @@ def encrypt_mock_data(mock_data: MutableMapping) -> str:
     return mock_data
 
 
+def get_schema_version(schema_folder, filename):
```

Comment:
We can maybe remove this function if we're setting `schema_version` with the file stem?

Reply:
Following on from a team discussion - we'll read this from the file contents to allow the `v3.json` to function with Runner. A follow-up ticket will be raised to move to the file name when we can add the unit data programmatically to the schema versions